packages/gatsby-theme-vtex/src/components/Minicart/Drawer/Delete.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, Spinner } from '@vtex/store-ui'
+import React, { useState } from 'react'
+
+import { useUpdateItems } from './useUpdateItems'
+
+type Props = {
+  index: number
+  variant?: string
+}
+
+const MinicartDelete = (props: Props) => {
+  const [isLoading, setIsLoading] = useState(false)
+  const updateItems = useUpdateItems(props.index)
+  const useDelete = {
```

Comment:
also, we could extract it into a file in sdk, what do you think ?

Reply:
I believe it is not necessary, since updateItems is already a SDK and the difference for useDelete is only the number of items that is passed as a parameter.